coding-concepts.js

```diff
@@ -7,67 +7,60 @@
 
 // --------------------1) What will this log?
 
-var cohort = "Alpha 2022"
-// console.log(cohort.split(""))
-
-// a) Your answer:
-// b) Verify and explain:
+var cohort = "Alpha 2022";
+console.log(cohort.split(""));
 
+// a) Your answer: ['A', 'L', 'P', 'H', 'A', ' ', '2', '0', '2', '2']
+// b) Verify and explain: The use of .split when console logging will return an array separated individually by character
```

Comment:
Separating a string into an array by character will also be dependent on the quotes inside the argument for split.  If there were no quotes inside the argument for split, would the answer change?